clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -995,7 +995,10 @@
     public static final Schema INIT_PRODUCER_ID_REQUEST_V0 = new Schema(
             new Field("transactional_id",
                     NULLABLE_STRING,
-                    "The transactional id whose pid we want to retrieve or generate.")
+                    "The transactional id whose pid we want to retrieve or generate."),
+            new Field("transaction_timeout",
```

Comment:
I think the field name should also include the time unit, so `transaction_timeout_ms`